style.css

```diff
@@ -93,10 +101,45 @@ nav {
 }
 
 /* ####### Hero Section ######## */
+
+@media only screen and (max-width: 576px) {
+  .navLinks.show {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    position: absolute;
+    left: 0;
+    top: 0;
+    height: 100vh;
+    width: 100vw;
+    border-bottom: 1px solid #dfe1e6;
+    padding-left: 30px;
+    padding-right: 42px;
+    padding-top: 26px;
+    background: rgba(97, 113, 255, 0.92);
+    text-align: center;
+    z-index: 2;
+    transition: 0.5s;
+    color: #fff;
+  }
```

Comment:
- [ Required ] I think it would be better if you add a blur effect to the mobile menu on your design, if you check the figma design you'll notice that it's very difficult to read what is on the background of the mobile menu. The blur effect prevents a user from focusing on the background and keeps focus on the menu items. Kindly see the screenshot below and rectify this.

## Example code
> I would like to suggest that you use the `backdrop-filter` CSS property to achieve this effect on your design. Please see the example below for more details on this.

```CSS
  .navLinks.show {
    display: flex;
    flex-direction: row-reverse;
    justify-content: space-between;
    position: fixed;
    left: 0;
    top: 0;
    height: 100vh;
    width: 100vw;
    border-bottom: 1px solid #dfe1e6;
    padding-left: 30px;
    padding-right: 42px;
    padding-top: 26px;
    background: rgba(97, 113, 255, 0.85);
    text-align: center;
    z-index: 2;
    transition: 0.5s;
    color: #fff;
    backdrop-filter: blur(3.5px);
  }
```

## Screenshot
| Your design | Figma design |
|-----------------|--------------------|
|![Screenshot from 2023-07-25 23-21-52](https://github.com/gilberthappi/portfolio/assets/16392046/55af050d-2b71-4569-b628-8fc77c1e7138)|![Template 1-Mobile-Menu](https://github.com/gilberthappi/portfolio/assets/16392046/25087619-b8e2-4091-b3d4-1ae45cdfb70b)

Reply:
Okey
thanks 
i understand